pubspec.yaml

```diff
@@ -61,6 +61,7 @@ dependencies:
   google_maps_flutter: ^2.1.1
   geolocator: ^8.2.0
   geocoding: ^2.0.2
+  json_schema2: ^2.0.2
```

Comment:
Is this package needed? Can we complete the current work without using this package?

Reply:
This package supports the _define a JSON schema and use it to validate the metadata set from the tmastr contract_ function. We cannot control what is inserted into the tmastr contract so it definitely needs validation.